include/ghostty.h

```diff
@@ -600,6 +600,7 @@ typedef enum {
   GHOSTTY_ACTION_COLOR_CHANGE,
   GHOSTTY_ACTION_RELOAD_CONFIG,
   GHOSTTY_ACTION_CONFIG_CHANGE,
+  GHOSTTY_ACTION_BELL,
```

Comment:
Feels like we should be more consistent with action names.

```
TOGGLE_SPLIT_ZOOM
PWD
RELOAD_CONFIG
BELL
```

Maybe `RING_BELL` would be more consistent?

Reply:
#7087 switched to RING_BELL